poet/__main__.py

```diff
@@ -1,6 +1,5 @@
-import sys
-
 from .poet import *
```

Comment:
Why move the sys import? I don't mind and I've seen this written a few ways but I'm not sure when to prefer which.

May as well just make this `from .poet import main` while we're here.